common/lib/xmodule/xmodule/js/spec/video/video_player_spec.js

```diff
@@ -380,22 +380,21 @@ function(VideoPlayer, HLS) {
             });
 
             describe('when the video is playing', function() {
-                beforeEach(function(done) {
+                beforeEach(function() {
                     state.videoPlayer.play();
+                });
 
+                it('call runTimer in seekTo on player', function(done) {
```

Comment:
@muhammad-ammar  I think the ticket was to fix "call updatePlayTime on player'" test. Do you think the changes that you've made in beforeEach will fix the test ?